import-automation/executor/test/integration_test.py

```diff
@@ -24,10 +24,11 @@
 NUM_LINES_TO_CHECK = 50
 
 CONFIGS = {
```

Comment:
maybe add some comment on where to find the auth (and thanks for figuring this out!)

Reply:
done